scripts/superdesk-archive/module.js

```diff
@@ -61,9 +63,21 @@
             $rootScope.$broadcast('multi:reset', {ids: ids}); // let react know
         };
 
+        /**
+         * update count on deselection
+         * e.g, when selected item gets published, corrected or killed
+         */
+        this.updateCount = function(itemId) {
+            _.remove(items, {_id: itemId});
+            this.count = items.length;
+        };
+
         // main
         this.reset();
         $rootScope.$on('$routeChangeStart', angular.bind(this, this.reset));
+        $rootScope.$on('multi:updateCount', function(_e, itemId) {
```

Comment:
imo it should be called `multi:remove` , it's not obvious if on `updateCount` message it should remove given item or add it..